wrappers/services.go

```diff
@@ -141,6 +169,20 @@ func AddSnapServices(s *snap.Info, inter interacter) (err error) {
 			return err
 		}
 		written = append(written, svcFilePath)
+
+		// Generate systemd socket files if needed
+		socketFiles, err := generateSnapSocketFiles(app)
```

Comment:
systemd creates the socket files by itself, so we don't need that.

Unfortunately that plus the earlier point that sockets are actually absolute paths creates a more serious issue that we'll need to find a solution for. We cannot allow snaps to create arbitrary files in arbitrary locations, which is what this is now doing.